src/content/1.7/development/components/global-components.md

```diff
@@ -0,0 +1,62 @@
+---
+title: Global JavaScript components
+weight: 70
+---
+
+# Global JavaScript components ?
+
+It's a [new system](https://github.com/PrestaShop/PrestaShop/blob/develop/admin-dev/themes/new-theme/js/app/utils/init-components.js) mainly created for module developers, which allow you to use components we already use in the core without importing it. [Here is the ADR](https://github.com/PrestaShop/ADR/blob/master/0009-expose-js-components-using-window-variable.md) where we decided what we should do to avoid this problem.
+
+You are now able to use components we expose to the `window.prestashop.component` object without importing anything.
+
+This object contains two attributes :
+
+- `component` which contain some components you can instantiate;
+- `instance` which contains every instances of initiated components.
+
+It allow module developers to avoid importing path such as `../../../../admin-dev/themes/new-module/js/components/translatable-field` because it was making hard to add a CI in order to build the module.
```

Comment:
```suggestion
It allows module developers to avoid importing path such as `../../../../admin-dev/themes/new-module/js/components/translatable-field` because it was making hard to add a CI in order to build the module.
```

allow's' right?